src/addons/plugin/device-proxy.js

```diff
@@ -27,8 +27,21 @@ class DeviceProxy extends Device {
       this.properties.set(propertyName, propertyProxy);
     }
 
+    // Copy over any extra device fields which might be useful for debugging.
+    this.deviceDict = {};
+    for (let field in deviceDict) {
+      if (field in ['id', 'name', 'type', 'properties', 'actions', 'events']) {
```

Comment:
I believe this should be:
`if (['id', 'name', 'type', 'properties', 'actions', 'events'].includes(field))`

Reply:
You are correct. I thought I tested this, but apparently I didn't.